packages/components/src/components/form/_form.scss

```diff
@@ -29,6 +29,7 @@
     // `flex-direction: row`
     flex: 1 1 auto;
     align-items: flex-start;
+    margin-bottom: $carbon--spacing-06;
```

Comment:
Would adding this to `bx--form-item` unexpected add margin for folks using this control? Might be good to offer it by default, but might catch folks off guard if we add this layout potentially

Reply:
yeah I think this is part of a discussion that's been on hold since the v10 effort began